### PR TITLE
GitHub workflow

### PR DESCRIPTION
**Pull request by ashleywang1**
# Description

This change kicks off the following workflow:

Gloo release v1.7.0-betax
-> Creates a branch in solo-apis called gloo-v1.7.0-betax
-> That branch is part of a PR
-> Someone from Solo has to approve and hit the "run workflow" button
-> Once the PR is merged into master, another github workflow runs that takes the branch name (gloo-v1.7.0-betax) and tags the latest master commit with that name.

See the test tag here: https://github.com/solo-io/solo-apis/releases/tag/gloo-vtest

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: .github/workflows/update-solo-apis.yaml [hunk 1]
```diff
@@ -41,12 +41,7 @@ jobs:
...
-            git commit -m "Sync Gloo APIs to ${{ github.event.release.tag_name }}"
-            git checkout -B update-gloo-apis
-            git remote add solo-apis git@github.com:solo-io/solo-apis.git
-            git push solo-apis update-gloo-apis
-          fi
\ No newline at end of file
+          git commit -m "Sync Gloo APIs to ${{ github.event.release.tag_name }}" --allow-empty
+          git checkout -B gloo-v${{ github.event.release.tag_name }}
+          git remote add solo-apis git@github.com:solo-io/solo-apis.git
+          git push solo-apis gloo-v${{ github.event.release.tag_name }}
```
Comment (saiskee) [inline .github/workflows/update-solo-apis.yaml:47]: Small nit but Can we DRY the branch name?

**Review by saiskee — DISMISSED** [review]
Looks good Thanks!!

**Review by npolshakova — APPROVED** [review]
Looks good!

**Pull request merged**